### ISSUE TITLE
manifest.ttl does not need to be minimal

### ISSUE THREAD

**Issue by falkTX** [sjaehn/lv2tutorial#1]
Hi, just starting a few corrections for things you have mentioned either on the 1st video or in written text.

First thing is that you say manifest.ttl has the entry point and minimal data, where the plugin real definitions are on a separate file.
While this practice is common, it is not required and everything will work fine if you only have a single manifest.ttl file and nothing else.
You can put as many resources as you want in separate files, you can even add new stuff to existing pre-installed plugins from a separate bundle that reference some other URI.

manifest.ttl is just the entry point as defined by LV2, but whatever is inside and where stuff is defined is up to each one to do as they see best.

PS: LV2 bundles also do not need .lv2 suffix, see setbfree as example.

**Comment by falkTX** [comment]
Correction, after more careful watching, you just give an overview of an existing plugin.
So this is not a real issue I guess, my apologies.

Might be good to mention that this structure is not a necessity.

**Comment by sjaehn** [comment]
Yes, there is no need for a minimal manifest.ttl. You can put all in there. And if you also put all code into a single binary, then you end up with the minimum of two files (manifest.ttl + plugin.so) as I mentioned. But it's a bad style. And performance may become a point especially if you have got a lot of non-`lv2:Plugin` manifest.ttls to be scanned.
